source/blog/2018-07-10-Skydive-With-oVirt.html.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Skydive With oVirt 
+author: mburman
+tags: oVirt, oVirt 4.2, open source virtualization,network
+date: 2018-07-10 09:01:00 UTC
+---
+
+## Summary
+
+Skydive is an open source real-time network topology and protocols analyzer providing a comprehensive way of understanding what is happening in your network infrastructure.
+The common use cases will be, troubleshooting, monitoring, SDN integration and much more.
+It has features such as:
+* Topology capturing - Captures network topology, interface, bridge and more
+* Flow capture - Distributed probe, L2-L4 classifier, GRE, VXLAN, GENEVE, MPLS/GRE, MPLS/UDP tunnelling support
+* Extendable - Support for external SDN Controllers or container based infrastructure, OpenStack, OpenContrail, Docker. Supports extensions through API
+* Distributed
+
+[Skydive network](http://skydive.network/) - Link to Skydive site
+
+## Installation steps
```

Comment:
The summary should be ended before the Installation steps. Maybe the keyword `READMORE` does the magic?

Reply:
Not sure i understand.
I removed the header summary as you suggested. What the READMORE stand for?